### DISCUSSION TITLE
How to Added Shortcodes/php function Breadcrumb in the theme?

### DISCUSSION THREAD

**Discussion by SL33pxxs** [flyntwp/flynt#509] [Q&A] [answered]
I Used Rank Math Plugin to added shortcodes/function Breadcrumb, how to display it in theme?
Thanks in Advanced

**Comment by timohubois** [comment, marked as answer]
@SL33pxxs: You can [call all PHP functions directly with timber](https://timber.github.io/docs/v2/guides/functions/) in combination with the native [do_shortcode](https://developer.wordpress.org/reference/functions/do_shortcode/) function:

`{{ function('do_shortcode', '[YOUR_SHORTCODE_HERE]') }}`

Or use the timber filter as @aaronmeder noticed here: https://github.com/flyntwp/flynt/discussions/509#discussioncomment-6810119

**Reply by SL33pxxs** [reply]
Thanks Very Work it!

**Comment by aaronmeder** [comment]
Hey @SL33pxxs 

An easy way to use shortcodes in Timber's Twig Templates is by using the shortcodes filter:
``{{ "[myshortcode]"|shortcodes }}``

See some other ways here:
[do_shortcode in Timber ](https://github.com/timber/timber/issues/744)

Is that what you were looking for?
Aaron

**Reply by SL33pxxs** [reply]
Thanks for explaint, work it!
